spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,206 @@
+require 'rails_helper'
+
+RSpec.describe 'Tasks', type: :system do
+  include LoginSupport
+
+  let(:user) { create(:user, email: 'user@example.com') }
+  let(:other_user) { create(:user, email: 'other_user@example.com') }
+  let(:user_task) { create(:task, title: 'user_task', user: user) }
```

Comment:
taskには必ずuserが関連付いているので、letの定義名はtaskで良い気がします。
```suggestion
  let(:task) { create(:task, title: 'user_task', user: user) }
```

Reply:
[85cc7ab](https://github.com/SakitaDaiki0623/sample_app_for_rspec/pull/3/commits/85cc7ab93ae33d7015a9678efca5b43b30401565)で修正しました。